requirements.txt

```diff
@@ -4,7 +4,7 @@ wheel
 # PyTorch
 # Temp. disabled the next line as it gets currently resolved to https://download.pytorch.org/whl/rocm3.8/torch-1.7.1%2Brocm3.8-cp38-cp38-linux_x86_64.whl
 # --find-links=https://download.pytorch.org/whl/torch_stable.html
-torch>1.5,<1.10
+torch==1.10.0
```

Comment:
Can we be a bit more flexible here and allow `torch>1.5,<1.11` ? 
Should work the same for colab and any fresh installation. Not sure if there are any other recent changes in torch that are not backwards compatible.

Reply:
Done :)